source/braille.py

```diff
@@ -2009,6 +2011,37 @@ def formatCellsForLog(cells: List[int]) -> str:
 """
 
 
+# The list containing the regions that will be shown in braille when the speak function is called
+# and the braille mode is set to speech output
+_regions: list[TextRegion] = []

```

Comment:
Could you give this a more descriptive name? e.g.:
```suggestion
_showSpeechInBrailleRegions: list[TextRegion] = []
```